client/src/components/layout/TopBar.js

```diff
@@ -16,6 +16,7 @@ const TopBar = ({ user }) => {
   ];
 
   const authenticatedListItems = [
+    <li>{user ? <img src={user.image} className="user-image"></img> : null}</li>,
```

Comment:
good work with the ternary, i just want the li to be a part of the ternary. If there isnt a user, this will render an empty li. Update so that no li is rendered at all unless the user is present